xarray/core/dataset.py

```diff
@@ -5549,6 +5703,79 @@ def dropna(
             Which variables to check for missing values. By default, all
             variables in the dataset are checked.
 
+        Example
+        -------
+
+        # Sample dataset with missing values
+        >>> data = {
+        ...     "time": [0, 1, 2, 3],
+        ...     "temperature": [25.0, None, 27.5, 28.0],
+        ...     "humidity": [60.0, 65.0, None, 70.0],
+        ... }
+        >>> dataset = xr.Dataset(data)
+        # Print the original dataset
+        >>> dataset
+        <xarray.Dataset>
+        Dimensions:      (time: 4, temperature: 4, humidity: 4)
+        Coordinates:
+        * time         (time) int64 0 1 2 3
+        * temperature  (temperature) object 25.0 None 27.5 28.0
+        * humidity     (humidity) object 60.0 65.0 None 70.0
+        Data variables:
+            *empty*
```

Comment:
This dataset doesn't have any data variables. Presumably you meant for humidity and temperature to be data variables, and time for be a coordinate?